bentoml/yatai/deployment/aws_lambda/lambda_app.py

```diff
@@ -62,18 +67,38 @@ def api_func(event, context):  # pylint: disable=unused-argument
     Application Load Balancer, in which case the parameter `event` must be type `dict`
     containing the HTTP request headers and body.
 
-    see: https://docs.aws.amazon.com/lambda/latest/dg/services-alb.html
+    You can find an example of which
+    variables you can expect from the `event` object on the AWS Docs, here
+    https://docs.aws.amazon.com/lambda/latest/dg/services-alb.html
     """
 
     if type(event) is dict and "headers" in event and "body" in event:
-        return bento_service_api.handle_aws_lambda_event(event)
+        logger.info('Got prediction request with body "%s"', {event["body"]})
+        prediction = bento_service_api.handle_aws_lambda_event(event)
+
+        logger.debug(
```

Comment:
following the comment about logging level above, I think here it should be updated to `logger.info`, and all others can be set to either `logger.debug` or `logger.error`

Reply:
Would line 76 and line 90 be logger.debug then?

Reply:
Actually, it's redundant information so I'll just remove those lines